Makefile

```diff
@@ -11,5 +11,16 @@ watch-fsharp:
 	@printf "\n\e[31mBuilding fsharp...\e[0m\n"
 	dotnet watch msbuild 
 
+format:
+	dotnet tool install fantomas
+	dotnet tool run fantomas src
 
+test-fsharp:
```

Comment:
This can be put in the `test` rule, no need for test-fsharp since the whole repo is F#

Reply:
Doing it this way shows this message `make: 'test' is up to date.` rather then running the tests. 
The tests would only run if I use `make-B test`. 
This could be because the test file is called "test" ?